client/src/Components/local/atoms/mapping-panel/helpers/has-mapping-permission.tsx

```diff
@@ -43,3 +43,5 @@ export const canMoveResize = (feature: Feature<Geometry, any>, permissions: Part
 export const canAddRemove = (feature: Feature<Geometry, any>, permissions: ParticipantMapping[]): boolean => {
   return hasMappingPermissions(feature, [MappingPermissions.AddRemove], permissions)
 }
+
+export const permissionError = (): void => alert('You do not have permission on this action')
```

Comment:
Hello @lebaphi - right now, I think this can just be a message to the console.  I can't quite think of a tidy UI for when the user tries to drag something they don't have permission for.